### DISCUSSION TITLE
HttpClient in Bootstrap

### DISCUSSION THREAD

**Discussion by SerratedSharp** [unoplatform/Uno.Wasm.Bootstrap#480] [Ideas]
I spent the weekend getting HttpClient/WasmHttpHandler to work in a Bootstrap project.

Adding Uno.UI.WebAssembly required I also include a AppManifest.js in the project.  The one thing that felt really dirty is I needed to manually declare the JS object for Uno.Http.HttpClient which would normally come from Uno.UI.js, since this file wasn't being emitted into the package_ folder.  I assume this is me doing something that isn't really supported and *.UI packages aren't intended to be used with Bootstrap alone.

Point being, I think it'd be nice if in the future HttpClient/WasmHttpHandler was moved to Bootstrap.  To me it's not really UI specific, and is very useful outside of Uno UI.  If I've got client side logic in WASM, I certainly prefer to make my API requests using HttpClient.  Take care.

**Comment by jeromelaban** [comment]
`WasmHttpHandler` is not needed anymore with .NET 6, the standard HttpClient is working properly and has more implemented features.
